src/main/java/org/neo4j/wrapper/LoggingService.java

```diff
@@ -19,35 +19,100 @@
  */
 package org.neo4j.wrapper;
 
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
 import java.io.File;
 import java.io.FileInputStream;
-import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.util.Properties;
+import java.util.logging.LogManager;
 
+/**
+ * The logging service mainly responsible for doing two things:
+ * <ul>
+ * <li>    Reset the property value of {@code java.util.logging.FileHandler.pattern} (which is defined in
+ * {@code windows-wrapper-logging.properties} file) to a absolute path if it is given as a relative path.
+ * The absolute path uses our default working directory {@code %NEO4J_HOME%} as parent.
+ * <li>    Create logging directories if not exists.
```

Comment:
There is a typo here:
"Create logging directories if they do not exist" or "Create logging directory if it does not exist"